Public/Src/Cache/ContentStore/Distributed/NuCache/CheckpointManager.cs

```diff
@@ -356,6 +367,36 @@ public Task<BoolResult> RestoreCheckpointAsync(OperationContext context, string
                 extraEndMessage: _ => $"CheckpointId=[{checkpointId}]");
         }
 
+        private void WriteLatestCheckpointToFile(OperationContext context, CheckpointState checkpointState)
+        {
+            try
+            {
+                _fileSystem.WriteAllText(_lastCheckpointFile, $"{checkpointState.CheckpointId},{checkpointState.CheckpointTime}");
+            }
+            catch (Exception e)
+            {
+                _tracer.Debug(context, $"Failed to write latest checkpoint state to disk: {e}");
+            }
+        }
+
+        public (string id, DateTime checkpointTime)? GetLatestCheckpointInfo(OperationContext context)

```

Comment:
>GetLatestCheckpointInfo [](start = 53, length = 23)

nit: checkpointId?

The naming looks a bit inconsistent. It should be either 'checkpointId and checkpointTime' or 'id' and 'time', imo.